crates/figma_import/src/figma_schema.rs

```diff
@@ -221,23 +200,7 @@ pub struct Hyperlink {
     #[serde(default)]
     pub node_id: String, // XXX: This is "nodeID" in Figma; we might not be deserializing ok...
 }
-// XXX ColorStop, Transform
-#[derive(Deserialize, Serialize, Debug, Clone, Default, PartialEq)]
-pub struct Vector {
-    pub x: Option<f32>,
-    pub y: Option<f32>,
-}
-impl Vector {
-    pub fn is_valid(&self) -> bool {
-        self.x.is_some() && self.y.is_some()
-    }
-    pub fn x(&self) -> f32 {
-        self.x.unwrap_or(0.0)
-    }
-    pub fn y(&self) -> f32 {
-        self.y.unwrap_or(0.0)
-    }
-}
+
```

Comment:
Are these types that are removed from figma_schema the same as the toolkit schema types of the same name? Per your other comments, maybe we should keep figma_schema the same and convert them to the legacy types (and later the proto-generated types).

Reply:
So I've put back Rectangle and Vector. I needed an into() for Rectangle, but I didn't find any places that needed Vector to be converted. I moved Color too, which wasn't in figma_schema but is used in the figma_schema types. I don't know if it makes sense to keep two copies of it though, it's such a basic type.